docs/population.md

```diff
@@ -343,3 +343,25 @@ To make use of the urban type, the following data is needed:
 - Put the downloaded *zip* file into `data/urban_type`, so you will have the file `data/urban_type/UU2020_au_01-01-2023.zip`
 
 Then, you should be able to run the pipeline with the configuration explained above.
+
+### Income
+
+This pipeline allows using the [Bhepop2](https://github.com/tellae/bhepop2) package for income assignation. 
+
+By default, Eqasim infers income from Filosofi global income distribution (on the relevant commune). 
+An income value is drawn from this distribution, independent of the household characteristics. This method is called
+`uniform`.
+
+Bhepop2 uses income distributions on subpopulations. For instance, Filosofi provides distributions depending on household size.
+Bhepop2 tries to match all the available distributions, instead of just the global one. This results in more
+accurate income assignation on subpopulations, but also on the global synthetic population.
```

Comment:
Could you add a link to the Bhepop2 documentation to give more details / or on the relevant paper?

Reply:
Done